protoc-gen-grpc-gateway/descriptor/grpc_api_configuration.go

```diff
@@ -0,0 +1,94 @@
+package descriptor
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/golang/protobuf/proto"
+	"google.golang.org/genproto/googleapis/api/annotations"
+)
+
+// GrpcAPIService represents a stripped down version of google.api.Service .
+// Compare to https://github.com/googleapis/googleapis/blob/master/google/api/service.proto

```

Comment:
Could you add a comment in here that the file you're avoiding importing has 23 import statements and that is why it's being avoided as well as a note that says "If a significant subset of these (>50%) are being reproduced in this file, swap it out for the generated version.